### DISCUSSION TITLE
使用paddleocr 身份证中的姓名无法识别

### DISCUSSION THREAD

**Discussion by Anxiangchegu** [PaddlePaddle/PaddleOCR#12378] [Q&A]
### 问题：使用paddleocr 身份证中的姓名无法识别。
如下图：
![识别不完全](https://github.com/PaddlePaddle/PaddleOCR/assets/30336323/cb56386e-a0ea-4b7d-b9bc-5b5629f7ca3d)

代码如下：
`
import paddle
from paddleocr import PaddleOCR
from paddleocr.tools.infer.utility import draw_ocr

print(paddle.__version__)

ocr = PaddleOCR(use_angle_cls=True, lang="ch")  # need to run only once to download and load model into memory

img_path = '../static/idCard.jpg'

result = ocr.ocr(img_path, cls=True)
print(result)
for idx in range(len(result)):
    res = result[idx]
    for line in res:
        print(line)
`

请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：Windows 11 家庭中文版22H2，华为笔记本
- 版本号/Version：Paddle：  PaddleOCR：paddle 2.6.0，paddleocr>=2.0.1，python 3.10

**Comment by tink2123** [comment]
paddleocr的版本是多少呢？ 建议版本号大于2.7.0， 还识别不了可以尝试更换检测模型为server
